config/site_settings.yml

```diff
@@ -1985,7 +1985,15 @@ developer:
   enable_safe_mode:
     default: true
     client: true
-  enable_experimental_sidebar:
```

Comment:
SiteSetting is renamed but realistically this is only used on our internal side so I'm not adding a migration to clean up the record.